test/pipeline-runs/go-tests-run.yaml

```diff
@@ -0,0 +1,33 @@
+# Pipelinerun for go unittests
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: go-test-
+  namespace: kf-ci
+  labels:
+    pipeline: go-test
+    repo: gcp-blueprints
+spec:
+  params:
+  # When running under prow this value will automatically be replaced 
+  # by the appropriate GCS location for the test.
+  # So the value here only matters for manual runs.
+  - name: artifacts-gcs
+    value: "gs://kubeflow-ci-deployment/gabrielwen-testing-2"
```

Comment:
minor:  Is there a more generic name can be used?

Reply:
I changed it to an empty place holder.